spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  let(:user1) {FactoryGirl.create(:user)}
+  context "GET #show/:id" do
```

Comment:
thêm line trống trước line này

phân biệt `let` và `let!` nào

Reply:
Dạ, phương thức let dùng để gọi 1 lần khi được gọi tới trong 1 khối it còn let! cho phép gọi lại phương thức trước mỗi it